lib/helper.js

```diff
@@ -170,10 +175,68 @@ async function deleteAllLabels(token, owner, host, repository, cli) {
 
     // Done
     if (cli.flags.uploadLabels) echo.success('Done!\n')
-    else {
+    else if (exit) {
         console.log()
-        echo.success('Finished!', true)
+        echo.success('Finished!', exit)
+    }
+}
+
+// Gets array of repository names in "owner" organization
+async function getRepositories(token, owner, host, cli) {
+    // Variables
+    let repos = []
+    let pageCount = 0
+    const perPage = 100
+    
+    // Check required Flags (repository not needed here but need to send so check won't fail)
+    checkRequiredFlags(token, owner, "sample")
+
+    // Get headers for a request to list all repositories for the given org
+    let response = await axios.headRepoList(true, token, owner, host, 1)
```

Comment:
Personal preference: `const response`
```suggestion
    const response = await axios.headRepoList(true, token, owner, host, 1)
```